homeassistant/components/ecovacs/strings.json

```diff
@@ -67,6 +67,26 @@
         "name": "Total time cleaned"
       }
     },
+    "select": {
+      "water_amount": {
+        "name": "Water amount",
+        "state": {
+          "high": "High",
+          "low": "Low",
+          "medium": "Medium",
+          "ultrahigh": "Ultrahigh"
```

Comment:
```suggestion
          "ultrahigh": "Ultra high"
```

Reply:
I searched the Internet and found more places where it is written together as separated.
https://www.dictionary.com/browse/ultrahigh